### PR TITLE
Ernst pda in omnibox

### PR DESCRIPTION
**Pull request by ernstkui**
Comes with https://github.com/nens/lizard-nxt/pull/1845 and https://github.com/nens/hydra-core/pull/312

image

### CODE REVIEW

**Comment by RoelvandenBerg** [comment]
When creating an Annotation the asset geometry is posted. Annotations cannot handle polygons so they break. Please fix annotations.

File: app/lizard-nxt-filters.js [hunk 1]
```diff
@@ -505,6 +505,7 @@ angular.module('lizard-nxt-filters')
       'overflow': gettext('Overflow'),
       'pipe': gettext('Pipe'),
       'pumpstation': gettext('Pump station'),
+      'pumpeddrainagearea': gettext('Pump drainage area'),
```
Comment (RoelvandenBerg) [inline app/lizard-nxt-filters.js]: **pump _ed_** drainage area?

Comment (ernstkui) [inline app/lizard-nxt-filters.js]: Hmm, I don't know

Comment (RoelvandenBerg) [inline app/lizard-nxt-filters.js]: Het is of pumped of pump niet beide toch?

**Comment by RoelvandenBerg** [comment]
It works

**Pull request merged**